index.html

```diff
@@ -681,133 +162,51 @@ <h1 class="jumbotron-heading display-1">Iris Project</h1>
-                <a href="https://iris-project.org/transfinite-iris/" class="btn btn-outline-primary btn-sm">website</a>
-                <a href="https://gitlab.mpi-sws.org/iris/transfinite" class="btn btn-outline-primary btn-sm">Coq development</a>
-              </div>
-            </div>
-          </dd>
-          <dt>RefinedC: A Foundational Refinement Type System for C Based on Separation Logic Programming</dt>
-          <dd>
-            <div class="entry">
-              Michael Sammler, Rodolphe Lepigre, Robbert Krebbers, Kayvan Memarian, Derek Dreyer, Deepak Garg<br/>
-              <small class="text-muted">Submitted for publication</small>
-              <div class="buttons">
-                <a href="https://plv.mpi-sws.org/refinedc/paper.pdf" class="btn btn-outline-primary btn-sm">[preprint] .pdf</a>
-                <a href="https://plv.mpi-sws.org/refinedc/" class="btn btn-outline-primary btn-sm">website</a>
-                <a href="https://gitlab.mpi-sws.org/iris/refinedc" class="btn btn-outline-primary btn-sm">Coq development</a>
-              </div>
-            </div>
-          </dd>
-
-          <dt>Actris 2.0: Asynchronous Session-Type Based Reasoning in Separation Logic</dt>
-          <dd>
-            <div class="entry">
-              Jonas Kastberg Hinrichsen, Jesper Bengtson, Robbert Krebbers<br/>
-              <small class="text-muted">Submitted for publication</small>
-              <div class="buttons">
-                <a href="pdfs/2020-actris2-submission.pdf" class="btn btn-outline-primary btn-sm">[preprint] .pdf</a>
-                <a href="https://gitlab.mpi-sws.org/iris/actris" class="btn btn-outline-primary btn-sm">Coq development</a>
-              </div>
-            </div>
-          </dd>
-
-          <dt>ReLoC Reloaded: A Mechanized Relational Logic for Fine-Grained Concurrency and Logical Atomicity</dt>
-          <dd>
-            <div class="entry">
-            Dan Frumin, Robbert Krebbers, Lars Birkedal<br/>
-              <small class="text-muted">Submitted for publication</small>
-              <br/>
-              <div class="buttons">
-                <a href="pdfs/2021-reloc-reloaded-submission.pdf" class="btn btn-outline-primary btn-sm">[preprint] .pdf</a>
-                <a href="https://iris-project.org/reloc/" class="btn btn-outline-primary btn-sm">website</a>
-                <a href="https://gitlab.mpi-sws.org/iris/reloc" class="btn btn-outline-primary btn-sm">Coq development</a>
+                {% if doc.pdf_url %}
+                <a href="{{ doc.pdf_url }}" class="btn btn-outline-primary btn-sm">.pdf</a>
+                {% endif %}
+                {% if doc.other_urls %}
+                {% for e in doc.other_urls %}
+                <a href="{{ e.url }}" class="btn btn-outline-primary btn-sm">{{ e.name }}</a>
+                {% endfor %}
+                {% endif %}
```

Comment:
Is there a way to avoid the code repetition for the three lists here? For my own website I solved this using [`_includes`](https://git.ralfj.de/web.git/tree/refs/heads/master:/ralf/_includes), which are used like `{% include menu-level.html menu=menu %}`.

Reply:
Yes, there is a way. However I don't think it is such a big deal for now, and we might want to experiment with the template (maybe have different things for the different lists), so I'd keep things as they are for now.

Another thing we'll want is to have a template for the whole page, and to write the contents using markdown, but I'd rather do that later as well.